### DISCUSSION TITLE
App does not exit when expected to exit

### DISCUSSION THREAD

**Discussion by dotancohen** [osmandapp/OsmAnd#13871] [Q&A]
### Description
When exiting the OsmAnd application by all generally-accepted means on Android, that is by closing all notifications and exiting the application from the OS Application Selector screen, OsmAnd remains running in the background with no other way to close it.

Sometimes I'll notice this is the case because I'll be walking in the hall at work and OsmAnd will suddenly decide to tell me to turn left! I'm not concerned that my phone is directing me into the women's restroom, but I am concerned that OsmAnd is running when I specifically took all reasonable actions to exit it.

### How to reproduce?
1. Use OsmAnd navigation.
2. Swipe closed the OsmAnd notifications and swipe closed the OsmAnd application in the OS Application Selector screen.
3. Hear some time later in the day OsmAnd's voice.

### Your Environment
OsmAnd Version: 4.0.9
Android/iOS version: Android 11
Device model: Samsung Note 10 Lite

**Comment by scaidermern** [comment]
You need to explicitly stop the navigation. Otherwise OsmAnd will continue running via a background process. Don't just close the app, stop the navigation first. This is intentional behavior, as far as I know.

**Reply by dotancohen** [reply]
Thank you. As a user this is unexpected - and annoying - behaviour. When I've closed an app, I expect it to be closed. Not "still working, but hidden".

What is the use case of the current behaviour?

**Reply by vshcherb** [reply]
This is system behavior, if app has background process with notification, it can't be stopped with system gesture, only if you click Force Stop

**Reply by dotancohen** [reply]
I see, thank you.

**Comment by vshcherb** [comment]
You can stop it in the notification (extend it and click stop)
